packages/ui/src/ui/pages/groups/GroupEditorDialog/GroupEditorDialog.tsx

```diff
@@ -68,14 +74,20 @@ class GroupEditorDialog extends React.Component<GroupsPageTableProps> {
             comment += `${responsiblesComment}`;
         }
 
-        return saveGroupData(
-            groupName,
-            membersToAdd,
-            membersToRemove,
-            responsiblesToAdd,
-            responsiblesToRemove,
-            comment,
-        ).then(() => {});
+        return this.props
+            .saveGroupData({
+                initialGroupName,
+                groupName,
+                membersToAdd,
+                membersToRemove,
+                responsiblesToAdd,
+                responsiblesToRemove,
+                comment,
+            })
+            .then(() => {
+                return this.props.fetchGroups();
```

Comment:
Here I actually don't know what to do: despite the fact that we wait for all promises in saveGroupData, we still might get an old list of groups here.

I don't want to add timeout, so maybe we can wait for something else?